tutorials/Python/00_Introduction/Introduction.ipynb

```diff
@@ -133,7 +135,8 @@
     "Whilst jupyter notebooks can be useful for demonstrating python code and prototyping new ideas, the main way in which to run python code is to execute a script. To do this, you need to write the python instructions within a file and execute it via the python interpretor.\n",
     "\n",
     "Let's write our first python script:\n",
-    "1. In a new terminal, open a text editor (e.g. gedit; type `gedit hello.py` in the terminal and press enter; the text editor will open in a new window).\n",
+    "\n",
+    "1. In a brand new terminal, open a text editor (e.g. gedit; type `gedit hello.py` in the terminal and press enter; the text editor will open in a new window).\n",
```

Comment:
Do they really need to open an editor from the terminal?

Reply:
Agreed this entire section kinda bothers me tbh, it's been error prone in the past. Any suggestions? Should we ditch it and I just show them within some slides in the class?